docs/self-hosted/deployment/docker.md

```diff
@@ -153,10 +153,6 @@ Update the images using the latest image from chatwoot. Run the `rails db:chatwo
 ## Running Rails Console
 
 ```
-# Find the name of the rails server container
-docker ps
-# access the shell inside the container. replace name if you are using a different name
-docker exec -it $(docker ps --filter name=root_rails_1 -q) /bin/sh
-# start rails console
-RAILS_ENV=production bundle exec rails c
+# Run this from within the working directory you `docker-compose up -d` in the context of
```

Comment:
Could you please improve the comment ?

Reply:
Did you have some direction, on what you think could improve?

"Run, assuming project name is the present working directory of the docker-compose.yml."

Is this clearer? I Guess it is doing a lot, it's just that it's easier to copy-paste.

Maybe remove the comment?

Reply:
Let's drop the comment for now.